src/getting-started/starting-stack.md

```diff
@@ -1,6 +1,61 @@
 # Starting the stack
 
-Within [arch-local] there is a [compose.yaml] file. This is a descriptor for the pre-configured multi-container definition of the components required for standing up a local development environment.
+## Choose a track
+There are 2 tracks that developers can take when using `arch-cli`:
+1. [Program-only]
+  - A single-node execution environment for testing program logic.
+2. [Complete dApp]
+  - A complete, multi-node execution environment with consensus for testing a full-stack web dApp.
+
+_If you aren't sure where to start, stick with [Program-only]!_
+
+## Program-only
+This spins up a lightweight validator that effectively serves the purpose of testing program deployment and functionality by simulating a single-node blockchain environment locally.
```

Comment:
Users still need to run `arch-cli init` to set up the configuration directory. This should always be the first step.